### PR TITLE
CC | Adapt to the containerd changes done on kata-containers repo

### PR DESCRIPTION
**Pull request by fidencio**
This will easily allow us to test against the forked and the upstream containerd.

As the tests had changed completely on main, and as the main branch should never ever see the forked containerd, we're doing this only for the CCv0 case.

### CODE REVIEW

File: .ci/install_cri_containerd.sh
Comment (fidencio) [reply .ci/install_cri_containerd.sh]: I was, indeed.

**Comment by stevenhorsman** [comment]
Sorry, I forgot to point out that https://github.com/kata-containers/tests/blob/6d7723a1ec99e85f968f6333979dd830a570bad2/integration/containerd/cri/integration-tests.sh#L51 will also need updating

**Comment by fidencio** [comment]
> Sorry, I forgot to point out that
> 
> https://github.com/kata-containers/tests/blob/6d7723a1ec99e85f968f6333979dd830a570bad2/integration/containerd/cri/integration-tests.sh#L51
> 
> will also need updating

Yeah, I just noticed with the test failing.  At least we know the change works, and what will need to be updated.

Let me grow this PR (and the one in the Kata Containers) a little bit so we avoid having to wait for this CI to finish to start doing the other work.

**Review by stevenhorsman — APPROVED** [review]
LGTM assuming the tests are passing

**Comment by fidencio** [comment]
None of the changes here should directly impact TDX / SEV / SNP CIs, thus I'm going ahead and merging this one to avoid delaying even more v0.8.0 release.

**Pull request merged**
